### PR TITLE
Update sample-data

### PR DESCRIPTION
**Pull request by asimonutti**
Updated sample-data with current sample-data generated on 2020-02-19. Also added a few additional scripts to automate the process of updating sample-data from sample-data generated by integration tests and stored within AWS.

### CODE REVIEW

File: sample-data/update_sample_events.sh [hunk 1]
```diff
@@ -0,0 +1,16 @@
...
+  exit 1
+fi
+
+date=$(date '+%Y-%m-%d')
+git checkout -b sample-data-update-${date}
+./update_avro_from_aws.sh $1 $2
+./convert_avro_files_json.sh
+git add .
+git commit -m "Updated sample-data from AWS"
+gh pr create -t "Update sample-data" -b "Updating sample-data from ${date}"
```
Comment (rogusdev) [inline sample-data/update_sample_events.sh]: Trivially, I recommend a trailing newline

**Review by rogusdev — APPROVED** [review]

File: sample-data/update_sample_events.sh (hunk 1 above)
Comment (asimonutti) [reply sample-data/update_sample_events.sh]: Added the trailing newline so it would consistent with all the other scripts.

**Review by rogusdev — COMMENTED** [review]
I am curious why there were 2 rows in the files before and only 1 row now

**Comment by asimonutti** [comment]
Asked Jay about that, when we run the integration tests it uploads two examples of the schema with differing ids. Previously we uploaded both of them to the currents-examples but the automation only grabs one.

**Comment by rogusdev** [comment]
You are most likely fine in this case, but as a general habit, I would discourage relying solely on my reviews before merging.  I am only so aware of the details in the code these days ;)

**Pull request merged**